client/src/App.js

```diff
@@ -1,17 +1,67 @@
-import React from "react";
-import { BrowserRouter, Route } from "react-router-dom";
-import LandingPage from "./pages/Landing";
+import React, { useEffect, useState } from "react";
 import SearchPage from "./pages/SearchPage";
-
-import "./App.css";
+import { MuiThemeProvider } from "@material-ui/core";
+import { BrowserRouter, Route, Redirect, Switch } from "react-router-dom";
+import { theme } from "./themes/theme";
+import SignUp from "./components/SignUp";
+import SignIn from "./components/SignIn";
+import ProtectedRoute from "./components/ProtectedRoute";
+import Explore from "./components/Explore";
 
 function App() {
-  return (
-    <BrowserRouter>
-      <Route exact path="/" component={LandingPage} />
-      <Route path="/search" component={SearchPage} />
-    </BrowserRouter>
-  );
+	const [loggedIn, setLoggedIn] = useState(false);
+
+	useEffect(() => {
+		checkLoggedIn();
+	});
+
+	const checkLoggedIn = () => {
+		if (localStorage.loggedIn) {
+			return setLoggedIn(true);
+		} else {
+			return setLoggedIn(false);
+		}
+	};
+
+	const handleModalExit = () => {
+		setLoggedIn(true);
+	};
+
+	return (
+		<BrowserRouter>
+			<MuiThemeProvider theme={theme}>
+				<Switch>
+					<Route exact path="/">
+						<SearchPage></SearchPage>
```

Comment:
Just leaving a note. We talked about it at standup but change these to `<SearchPage />`